packages/dd-trace/src/config.js

```diff
@@ -9,6 +9,7 @@ const coalesce = require('koalas')
 const tagger = require('./tagger')
 const { isTrue, isFalse } = require('./util')
 const uuid = require('crypto-randomuuid')
+const { GIT_REPOSITORY_URL, GIT_COMMIT_SHA } = require('./plugins/util/tags')
```

Comment:
It looks like these are expected to be set by users? If that's the case they should probably be moved to `ext/tags`.

Reply:
yeah that makes sense. I'll do this in another PR though, because there are multiple files to change and I don't want to make this harder to review